storage/s3/build.gradle

```diff
@@ -27,7 +27,14 @@ ext {
 dependencies {
     implementation project(":storage:core")
 
-    implementation "com.amazonaws:aws-java-sdk-s3:$awsSdkVersion"
+    // Take out jackson libraries out to avoid collision with core jackson dependencies
+    implementation ("com.amazonaws:aws-java-sdk-s3:$awsSdkVersion") {
+        exclude group: "com.fasterxml.jackson.core"
+        exclude group: "com.fasterxml.jackson.dataformat"
```

Comment:
We still can merge but I wonder if there is a proper way in Gradle to enforce versions of transitive dependencies similar to maven dependencyManagement section. That would allow to solve this kind of issues once and for all. And in this case if there will be more dependencies bringing jackson as transitive we will have to add more exclusions.

Reply:
Agree, we should look for some way to do this from Gradle. Will have a look.

Reply:
I tried to add a custom task to handle this validation. Have a look, thank!